pkg/vm/engine/disttae/tools.go

```diff
@@ -966,7 +966,7 @@ func genBlockMetas(rows [][]any, columnLength int, fs fileservice.FileService, m
 					mp[blk.BlockID] = blockInfos[i]
 				}
 			} else {
-				mp[blk.BlockID] = blockInfos[i]
+				mp[blockInfos[i].BlockID] = blockInfos[i]
```

Comment:
I don't know, but line 966 looks really fishy ...

Reply:
i think it's ok.